resources/lib/categories.py

```diff
@@ -81,5 +81,5 @@ def make_category_list(url):
 
         xbmcplugin.endOfDirectory(handle=int(sys.argv[1]), succeeded=ok)
         xbmcplugin.setContent(handle=int(sys.argv[1]), content='episodes')
-    except Exception:
-        utils.handle_error()
+    except Exception as e:
+        utils.handle_error('Unable to build categories list', exc=e)
```

Comment:
Are you sure passing this is required? I have a feeling we get the exception by calling exc_info int he handling code, but just want to confirm

Reply:
You're right, I thought I saw the change in one of your commits from adding aussieaddonscommon to one of the other addons so I assumed it was needed, didn't check myself. But now I can see the error of my ways. `Error Contents: handle_error() takes exactly 1 argument (0 given)` in the issue reporter log would have made it easy for me but I missed it.